rinja/src/html.rs

```diff
@@ -120,6 +110,224 @@ const TABLE: Table = {
 const ESCAPED_BUF_INIT: [u8; 8] = *b"&#__;\0\0\0";
 const ESCAPED_BUF_LEN: usize = b"&#__;".len();
 
+#[derive(Debug, Clone, Copy)]
```

Comment:
Please just add a comment explaining why we need this type.

Reply:
I added an explanation, and reduced the code to the actually needed variants. Should be much easier to follow, now.

Reply:
Oh nice, thanks!

Reply:
Oh nice, thanks!